tutorial/part2/chapter6.md

```diff
@@ -380,5 +378,4 @@ The above code will generate the following app:
 
 ## Summary
 In this chapter, we have explored several options to upload data into a pandas dataframe that will be used inside a Dash app. We went through some basic data wrangling techniques that prepare our data for usage by Dash components.
-
```

Comment:
I like the last example just above the summary section but I think it might be too advanced to introduce a chanined callback and `prevent_initial_call` all at once. Could we break this down into 2-3 steps?